tests/OmniSharp.Roslyn.CSharp.Tests/FindReferencesFacts.cs

```diff
@@ -549,7 +549,7 @@ public class Foo
             Assert.Null(exception);
         }
 
-        [Fact]
+        [Fact(Skip = "Need Roslyn EA for the TestGeneratorReference")]
```

Comment:
The change to include checksum in the caching causes the test to throw due to our TestGeneratorReference class. An interface was added for testing purposes and we can use that once Roslyn exposes it to us through our ExternalAccess library.